### ISSUE TITLE
Wrong Vagrant $PATH after installation

### ISSUE THREAD

**Issue by ankushagarwal** [GoogleCloudPlatform/kubernetes#7280]
Ran this on my Mac OS X Yosemite
```
mkdir /Users/ankushagarwal/vagrant/kubernetes
cd /Users/ankushagarwal/vagrant/kubernetes
export KUBERNETES_PROVIDER=vagrant
curl -sS https://get.k8s.io | bash
```

After installation is complete I get the message : 
```
Kubernetes binaries at /Users/ankushagarwal/vagrant/kubernetes/kubernetes/kubernetes/cluster/
You may want to add this directory to your PATH in $HOME/.profile
Installation successful!
```

The binaries are at `/Users/ankushagarwal/vagrant/kubernetes/kubernetes/cluster/` and not at `/Users/ankushagarwal/vagrant/kubernetes/kubernetes/kubernetes/cluster/`

**Comment by erictune** [comment]
I'm out the next 2 weeks.  Un-assign myself in case anyone else has time to look at this.

**Comment by roberthbailey** [comment]
@derekwaynecarr for vagrant
@bentheelder in case you are on a mac (since I know you are using vagrant).

**Comment by roberthbailey** [comment]
It looks like this was fixed by #7310 so I'm going to close. If you run across this again please re-open this issue.

**Comment by BenTheElder** [comment]
@roberthbailey I'm primarily on a macbook with vagrant (I also use fedora 21 on a desktop); good to see it's already been fixed.
